### ISSUE TITLE
Add NullableContextAttribute

### ISSUE THREAD

**Issue by terrajobst** [dotnet/corefx#38460]
Related to #36222

@jcouv, what's the shape for this one?

**Comment by jcouv** [comment]
Below is the shape for `NullableContextAttribute` and `NullableMembersAttribute` (copied from https://github.com/dotnet/roslyn/pull/36152).

From our discussion this morning, I assume the decision for `NullableAttribute` applies to those as well. Namely, we'll let the compiler inject those types for now.

```C#
namespace System.Runtime.CompilerServices
{
    [System.AttributeUsage(
        AttributeTargets.Module |
        AttributeTargets.Class |
        AttributeTargets.Delegate |
        AttributeTargets.Interface |
        AttributeTargets.Method |
        AttributeTargets.Struct,
        AllowMultiple = false,
        Inherited = false)]
    public sealed class NullableContextAttribute : Attribute
    {
        public readonly byte Flag;
        public NullableContextAttribute(byte flag)
        {
            Flag = flag;
        }
    }

    public enum NullableMembers
    {
        Public = 0,   // public and protected only
        Internal = 1, // public, protected, internal
        All = 2,
    }

    [System.AttributeUsage(AttributeTargets.Module, AllowMultiple = false)]
    public sealed class NullableMembersAttribute : Attribute
    {
        public readonly NullableMembers Members;
        public NullableMembersAttribute(NullableMembers members)
        {
            Members = members;
        }
    }
}
```

FYI @cston

**Comment by terrajobst** [comment]
Closing, because we closed #36222 too.
